app/controllers/buyers/bulk_base_controller.rb

```diff
@@ -2,22 +2,46 @@
 
 class Buyers::BulkBaseController < FrontendController
   before_action :authorize_bulk_operations
+  before_action :initializa_errors, only: :create
+
+  def new; end
+
+  def create
+    @errors = nil
+
+    recipients.each do |recipient|
+      message = current_account.messages.build send_emails_params
+      message.to = recipient
+
+      @errors << message unless message.save && message.deliver!
```

Comment:
`@errors` supposed to be an array but I see it initialized as `nil`, how would `<<` work with this? Can we have some test to make sure errors do work?